src/cluster_slot_stats.c

```diff
@@ -47,12 +47,13 @@ static int markSlotsAssignedToMyShard(unsigned char *assigned_slots, int start_s
 }
 
 static uint64_t getSlotStat(int slot, slotStatType stat_type) {
+    kvstoreDictMetadata *meta = (kvstoreDictMetadata *)kvstoreGetDictMetadata(server.db->keys, slot);
```

Comment:
we can create another getter function to make it more readable :

```
static inline kvstoreDictMetadata *getSlotMeta(int slot, int createIfNeeded) {
    return (kvstoreDictMetadata *)kvstoreGetDictMeta(server.db->keys, slot, createIfNeeded);
}
````
There are 8 places in this file that can be simplified.

Reply:
Done.
Should I follow up with switching from dictMeta -> slotMeta throughout?